src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -146,6 +146,16 @@ public void ClearBoundBreakpoints()
             }
         }
 
+        public void SetError(AD7ErrorBreakpoint bpError, bool sendEvent = false)
```

Comment:
Is `sendEvent` ever set to `false`?

I see lots of usages of `_BPError` in which it's either just set (e.g., in `CanBind()`) and others in which it is sent.

Maybe you want to use `SetError` across this class in the existing code?